jwst/ramp_fitting/ramp_fit_step.py

```diff
@@ -240,7 +241,11 @@ def process(self, input):
             image_info, integ_info, opt_info, gls_opt_model = ramp_fit.ramp_fit(
                 input_model, buffsize,
                 self.save_opt, readnoise_2d, gain_2d, self.algorithm,
-                self.weighting, max_cores, dqflags.pixel)
+                self.weighting, max_cores, dqflags.pixel, suppress_one_group=self.suppress_one_group)
+
+            print("-" * 80)
```

Comment:
don't want to leave these in the final product

Reply:
I removed these.